src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -69,10 +69,6 @@ struct KeyValueViewModel {
                 // Convert into String in generated code
                 self.value = "String(\(param.name))"
             }
-        } else if key == "endpoint" {
-            self.value = "baseUrl.absoluteString" // baseUrl is a property of the generated client code
-            self.optional = false
-            self.paramName = nil
```

Comment:
Why did this go away?

Reply:
The service endpoint is from the 'baseUrl' which is initialized in the initializer of the Client code. The whole url will be resolved in ` guard let url = self.url(forTemplate: urlTemplate, withKwargs: pathParams) ` in the generated code. 

So I don't see we need to above binding between endpoint and baseUrl. 

In fact, the old implementation urltemplate ="{endpoint}{path}" will resolved in a wrong url. With the old implementation, the url was 'http://myserverhttp://myserver/{the path}'. so the {endpoint} is redundant that is why I remove it.